src/codegen/opt/build_cuda_on.cc

```diff
@@ -26,11 +30,65 @@ namespace codegen {
     }                                                                   \
   }
 
-std::string NVRTCCompile(const std::string& code) {
+
+std::string FindCUDAIncludePath() {
+#ifdef _WIN32
+  const std::string delimiter = "\\";
+#else
+  const std::string delimiter = "/";
+#endif
+  std::string cuda_include_path;
+  const char* cuda_path_env = std::getenv("CUDA_PATH");
+  struct stat st;
+  if (cuda_path_env != nullptr) {
+    cuda_include_path += cuda_path_env;
+    cuda_include_path += delimiter + "include";
+    return cuda_include_path;
+  }
+
+  cuda_include_path = "/usr/local/cuda/include";
+  if (stat(cuda_include_path.c_str(), &st) == 0) {
```

Comment:
stat function may not available in some of MSVC, consider only use stat query in linux and force user to set CUDA_PATH otherwise

Reply:
see https://github.com/dmlc/tvm/blob/master/src/runtime/threading_backend.cc#L14

Reply:
c.f. https://msdn.microsoft.com/en-us/library/14h5k7ff.aspx Sometimes the stat function is not available, and need to use _stat instead. Given the proposed path do not work for windows anyway, let us just skip this in windows